sopel/trigger.py

```diff
@@ -315,6 +332,14 @@ class Trigger(unicode):
     example, when setting ``mode -m`` on the channel ``#example``, args would
     be ``('#example', '-m')``
     """
+    urls = property(lambda self: self._pretrigger.urls)
+    """A tuple containing all URLs found in the text message.
+
+    :type: tuple
+
+    These are the URLs found in the text message of the IRC line, usually after
+    the ` :` section.
```

Comment:
```suggestion
    These are the URLs found in the last parameter of the IRC line.
```

Since you decided not to be smart, and just always run `search_urls()` on `self.args[-1]`, there's no need to even mention <tt>&nbsp;:</tt> at all.

The last parameter also isn't necessarily always the text message component, e.g. for a kick/ban with no message.